sdv/cag/base.py

```diff
@@ -19,6 +21,8 @@ def __init__(self):
         self.metadata = None
         self._fitted = False
         self._single_table = False
+        self._dtypes = None
+        self._constraint_col_formatters = {}
```

Comment:
nitpick: you can honestly just call these self._formatters. We're in a constraint so that part is unnecessary and the column part I could take or leave

Reply:
Yes I agree, done in [b01a388](https://github.com/sdv-dev/SDV/pull/2557/commits/b01a388c78c3cc2d40f0afa0a73413b3f00737af)